litmus-portal/frontend/README.md

```diff
@@ -1,46 +1,21 @@
-[![Docker Pulls](https://img.shields.io/docker/pulls/litmuschaos/ansible-runner.svg)](https://hub.docker.com/r/litmuschaos/ansible-runner)
-[![GitHub stars](https://img.shields.io/github/stars/litmuschaos/litmus?style=social)](https://github.com/litmuschaos/litmus/stargazers)
-[![GitHub issues](https://img.shields.io/github/issues/litmuschaos/litmus)](https://github.com/litmuschaos/litmus/issues)
-[![Twitter Follow](https://img.shields.io/twitter/follow/litmuschaos?style=social)](https://twitter.com/LitmusChaos)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3202/badge)](https://bestpractices.coreinfrastructure.org/projects/3202)
-[![BCH compliance](https://bettercodehub.com/edge/badge/litmuschaos/litmus?branch=master)](https://bettercodehub.com/)
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus?ref=badge_shield)
-[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
-<br><br>
-
-<img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fpmcdeadline2.files.wordpress.com%2F2016%2F09%2Ftwitter-logo.jpg&f=1&nofb=1" alt="LitmusChaos Twitter" width="12"> (https://twitter.com/LitmusChaos/) **Follow us on Twitter!** [**@LitmusChaos**](https://twitter.com/LitmusChaos/)
-
-<a href="https://litmuschaos.io"><img src="https://i.ibb.co/QKSMq9g/Screenshot-from-2020-09-14-18-54-05.png" align="right" width="250px" /></a>
-
-## **Learn about Litmus** 😮😎
-
+Learn about Litmus 😮😎
 Litmus is a toolset to do cloud-native chaos engineering. Litmus provides tools to orchestrate chaos on Kubernetes to help SREs find weaknesses in their deployments. SREs use Litmus to run chaos experiments initially in the staging environment and eventually in production to find bugs, vulnerabilities. Fixing the weaknesses leads to increased resilience of the system.
-<br><br>
 
-## Table of Contents
 
-1. [Project Structure Practices](#1-project-structure-practices)
-2. [Available Scripts](#2-available-scripts)
-3. [Tech Stack](#3-tech-stack)
-4. [Frontend Contribution Guidelines](#4-frontend-contribution-guidelines)
-5. [Error Handling Practices](#5-error-handling-practices)
-6. [Testing And Overall Quality Practices](#6-testing-and-overall-quality-practices)
-7. [Project Structure Practices](#7-project-structure-practices)
-8. [Coding Style Guide](#8-coding-style-guide)
-9. [User Guide for Litmus Portal](#9-user-guide-for-litmus-portal)
-   <br><br><br>
+Table of Contents
+Project Structure Practices
+Available Scripts
+Tech Stack
+Frontend Contribution Guidelines
+Error Handling Practices
+Testing And Overall Quality Practices
+Project Structure Practices
+Coding Style Guide
+User Guide for Litmus Portal
```

Comment:
This removes the Markdown styling and links based on table content, thus we would prefer having the previous one